### DISCUSSION TITLE
Viewing JetStream

### DISCUSSION THREAD

**Discussion by alberk8** [nats-io/nats.net#529] [Q&A]
I have Jetstream setup with a subject.  There is already a consumer for that.   What is the correct way to just view that stream (in .Net) with disturbing the existing consumer?

**Comment by Jarema** [comment]
Use the Ordered Consumer, or any new consumer that can be deleted after walking through the whole stream.


It will not affect other consumers unless it is a workqueue stream. 
If that is the case, you need to wait for nats 2.11 batch get feature or use direct get for every sequence.

**Reply by alberk8** [reply]
It is not in a workqueue yet.  Need to study more on all the type of Consumer :)
